src/config/Config.zig

```diff
@@ -4338,7 +4338,9 @@ pub const ColorList = struct {
             count += 1;
             if (count > 64) return error.InvalidValue;
 
-            const color = try Color.parseCLI(raw);
+            // Trim whitespace from each color value
+            const trimmed = std.mem.trim(u8, raw, " \t");
+            const color = try Color.parseCLI(trimmed);
```

Comment:
I know it's a bit obvious but please add a unit test, it just helps prevent regressions in the future. 😄

Reply:
Sorry, I did write tests for it, but I forgot to push it up and hurried off to work on #5926. 🫠